custom_components/tuya_local/devices/morray_level_sensor.yaml

```diff
@@ -0,0 +1,109 @@
+name: Level sensor
+products:
+  - id: ywcgq
```

Comment:
Product id is not usually this short, even for Zigbee devices. Is it missing some characters, or a different id?